4337/test/eip4337/EIP4337ModuleExisting.spec.ts

```diff
@@ -27,28 +22,7 @@ describe('Safe4337Module - Existing Safe', () => {
     }
   })
 
-  describe('getOperationHash', () => {
```

Comment:
Moved this to the general `EIP4337Module` test suite, IMO test suite for existing safes wasn't the right place